charts/port-agent/templates/deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
-        {{- include "port-agent.selectorLabels" . | nindent 8 }}
```

Comment:
I don't understand why remove the selector and add labels?
We might need both..

Reply:
its inside of it